src/oncall/scheduler/default.py

```diff
@@ -90,7 +90,7 @@ def get_busy_user_by_event_range(self, user_ids, team_id, events, cursor, table_
 
         query = '''
                 SELECT DISTINCT `user_id` FROM `%s`
-                WHERE `user_id` in %%s AND (%s) AND (%s)
+                WHERE `user_id` in %%s AND (%s) AND (%s or `role_id`=5)
```

Comment:
Can we make the hardcoded `5` a lookup subquery against the `role` table? We can't guarantee that the role ID will always be a 5 in every situation.